pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -562,6 +562,12 @@ func (c *Cluster) provisionPodTemplateSpec(osdProps osdProperties, restart v1.Re
 	udevVolume := v1.Volume{Name: "udev", VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: "/run/udev"}}}
 	volumes = append(volumes, udevVolume)
 
+	// If not running on PVC we mount the rootfs of the host to validate the presence of the LVM package
+	if !osdProps.onPVC() {
+		rootFSVolume := v1.Volume{Name: "rootfs", VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: "/"}}}
```

Comment:
This seems like a pretty big security hole that I hope isn't necessary. Could we instead try to mount `/sbin` and `/usr/sbin` individually? I think the better thing would be to mount using a `subPath` to attempt to mount the lvm binary itself individually, but that could potentially crash the pod if the binary isn't there.

Reply:
Also, what if a user has a PSP that does not allow mounting host paths?

Reply:
@BlaineEXE we primarily need to mount `/` so we can access `/rootfs`, directly trying mounting the whole path never worked for me (because it's pseudo fs?). So just lvm potential path won't be sufficient.

@andrewrynhard mounting host paths is already required for the monitor store, so unless you run on PVC (on which this code never triggered) then you cannot possibly deploy a cluster. Allowing hostpath mount for clusters, not on PVC is already a pre-requisite.